app/models/pet.rb

```diff
@@ -95,7 +95,7 @@ def self.org_pets(staff_org_id)
 
   # all pets under an organization with applications and no adoptions
   def self.org_pets_with_apps(staff_org_id)
```

Comment:
For some reason, when I test this locally, the user model is nil for AdopterAccount with id = 3. So, Kamala and Brandon both load fine in the applications list, but Bad does not. I am not sure why, but want to check if this is happening to you also before I spend more time on it. I have tried re-running seeds multiple times just to be sure.

Reply:
PR merged. https://github.com/rubyforgood/pet-rescue/pull/340